packages/angular/common/src/directives/navigation/tabs.ts

```diff
@@ -142,7 +178,45 @@ export abstract class IonTabs implements AfterContentInit, AfterContentChecked {
     }
   }
 
+  private setActiveTab(tab: string): void {
+    const tabs = this.tabs;
+    const selectedTab = tabs.find((t: any) => t.tab === tab);
+
+    if (!selectedTab) {
+      console.error(`tab with id: "${tab}" does not exist`);
```

Comment:
```suggestion
      console.error(`[Ionic Error]: Tab with id: "${tab}" does not exist`);
```

Will match existing log levels that we do (e.g. `[Ionic Warning]`). 

I would consider adding more context to make the error exceptionable. 

For example:
> [Ionic Error]: Unable to select tab with id: "foo". Verify that the element is present in the DOM.